<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(73, 81, 116);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="4" column="1">
      <widget class="QPushButton" name="contrast_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Контрастность</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="color_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Наложение цвета</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="negative_check">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Негатив</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="save_as_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(163, 163, 163);</string>
       </property>
       <property name="text">
        <string>Сохранить как</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="cancel_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 130, 130);</string>
       </property>
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="blur_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>235</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Размытие</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="sharp_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Резкость</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="quantize_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Уменьшить число цветов</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Line" name="line">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(185, 93, 107);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="save_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(163, 163, 163);</string>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="brightness_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Яркость</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="bw_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Черно-белое</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="files_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(62, 62, 62);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Недавно изменённые файлы</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="13">
      <widget class="QLabel" name="image">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
